util/xrpcutil/xrpcutil.go

```diff
@@ -0,0 +1,101 @@
+package xrpcutil
```

Comment:
can we put this in the `xrpc` package as `AutoClient` or something? Feels weird over here in `util`

Reply:
It can be a subpackage of the `xrpc` package but it can't be in the main `xrpc` package because of the dependency loop.